en/docs/develop/api-policies/create-a-policy.md

```diff
@@ -0,0 +1,291 @@
+# Create a Policy
+
+You can use policies to enforce some business logic on the Request, Response, or Fault flow of an API invocation. Using policies, you can make API invocations undergo slight behavioral modifications before reaching the backend. Similarly, you can modify the API response a client receives. In Choreo, you can add an API Policy only to an API Proxy. 
+
+A policy can be implemented as a Ballerina project and attached to an API Proxy. Follow the steps below to create a policy. 
+
+## Prerequisites
+
+1. Set up [Ballerina 2201.2.0 or later](https://ballerina.io/learn/install-ballerina/set-up-ballerina/).
+2. Open the ~/.ballerina/Settings.toml file and check if you have configured an access token to the Ballerina Central. If an access token is not configured in the file, follow the steps below: 
+    1. Generate a token from [https://central.ballerina.io/dashboard?tab=token](https://central.ballerina.io/dashboard?tab=token).
+    2. Download the generated Settings.toml file and copy it to your local ~/.ballerina directory.
+    
+Alternatively, you can set the access token through the **BALLERINA_CENTRAL_ACCESS_TOKEN** environmental variable.
+
+``` 
+export BALLERINA_CENTRAL_ACCESS_TOKEN=<access-token> 
+```
+
+##  Step 1: Initializing a Ballerina project
+
+A mediation policy is a Ballerina project. You can use the template provided by Choreo to initialize a mediation policy project with all the required configurations as follows:
+
+-  Create a Ballerina project for the mediation policy using the **mediation.template** as the project template.  
+   
+    **Format:**
+    ```
+    bal new -t choreo/mediation.template:1.0.0 <policy-name> 
+    ```
+    
+    **Example:**
+    ```
+    bal new -t choreo/mediation.template:1.0.0 validateHeader 
+    ```
+ The Ballerina project you generated above has the following content: 
+
+![Ballerina project](../../assets/img/develop/api-policy/ballerina-project.png){.cInlineImage-small}
+
+You can modify the Ballerina.toml and the Package.md files of the generated project as needed. For example, the **org**, the **package**, the package **version**, API documentation content, **keywords**, etc.
+
+![Ballerina toml](../../assets/img/develop/api-policy/ballerina-toml.png){.cInlineImage-small}
+
+## Step 2: Implement the policy
+
+Let's implement a policy to validate an incoming header in the request and the response. In the request path, if the request header is not present or if the validation fails, let's log an error and return a 403 Bad Request response to the client whereas in the response path, let's only log a message.
+
+Follow the steps below to implement the policy:
+
+- Open the policy.bal file in the Ballerina project and fill in/modify the generated policy stubs(request, response, or fault) to suit your requirements. 
+
+### Request Flow
```

Comment:
Having sub-topics makes it easier for navigation through the page TOC>

Reply:
At the moment it appears as if the sentence about the fault flow, the note and the tip all belong to the **Response flow** sub topic. 

In that case, shall we also have the **Fault flow** sub heading before the sentence that says we will not be making changes there? Also it would be better if the note and the tip are grouped under some other sub topic too.

Reply:
Done. Converted it to a section, and removed the tip.

Reply:
- Change the "Best Practices" title to "Best practices"
- The bullet points in this section are over-indented and they appear in a different format in the preview.

Reply:
done